README.md

```diff
@@ -13,6 +13,7 @@ A set of Grafana dashboards and Prometheus alerts for Kubernetes.
 | release-0.2  | v1.14.1 and before | v2.11.0+  |
 | release-0.3  | v1.17 and before   | v2.11.0+  |
 | release-0.4  | v1.18              | v2.11.0+  |
+| release-0.5  | v1.19              | v2.11.0+  |
```

Comment:
if `master` previously was `v1.18+`, why is release-0.5 v1.19 now?

Reply:
whoopsie, updated master to point to 1.19, great catch :+1:

Reply:
I am not sure things changed that we need to not support 1.18? Were there any metric changes? Tests seem to still pass?

Reply:
@lilic @brancz adapted to `v1.18+` only, PTAL, I guess that's what you meant.

Reply:
so why release a new version and not push changes to release-0.4?

Reply:
We do this when a new version of k8s is cut, so now that 1.19 is cut. So its more like 1.18 and 1.19 and previous versions might not work for 1.19. Hope that makes sense?